README.md

```diff
@@ -0,0 +1,286 @@
+| AOI    | Sets the AOI flag (see **Inertial Interrupt Modes** below) |
+| SIX_D  | Sets the 6D flag (see **Inertial Interrupt Modes** below) |
+
+#### Inertial Interrupt Modes
+
+The following is taken from the from [LIS3DH Datasheet](http://www.st.com/st-web-ui/static/active/en/resource/technical/document/datasheet/CD00274221.pdf) (section 8.21):
+
+| AOI | 6D  | Interrupt Mode                     |
+| --- | --- | ---------------------------------- |
+|  0  |  0  | OR combination of interrupt events |
+|  0  |  1  | 6 direction movement recognition   |
+|  1  |  0  | AND combination of events          |
+|  1  |  1  | 6 direction position recognition   |
+
+**Movement Recognition (01)**: An interrupt is generate when orientation move from unknown zone to known zone. The interrupt signal stay for a duration ODR.
+
+**Direction Recognition (11)**: An interrupt is generate when orientation is inside a known zone. The interrupt signal stay until orientation is inside the zone.
+
+### configureFreeFallInt(*state, [threshold, duration]*)
+The *configureFreeFallInt* configures the intertial interrupt generator to generate interrupts when the device is in free fall (acceleration on all axis appraoches 0). The default `threshold` is 0.5 Gs.The default `duration` is 5 sample.
+
+```Squirrel
+accel.configureFreeFallInt(true);
+```
+
+**Note:** This method will overwrite any settings configured with the *configureInertialInt*.
+
+### configureClickInterrupt(*state, [clickType, threshold, timeLimit, latency, window]*)
+
+Configures the Click Interrupt Generator:
+
+| parameter | type       | default                    | description                                              |
+| --------- | ---------- | -------------------------- | -------------------------------------------------------- |
+| state     | bool       | n/a                        | `true` to enable, `false` to disable                     |
+| clickType | CONST      | LIS3DH.SINGLE_CLICK        | LIS3DH.SINGLE_CLICK or LIS3DH.DOUBLE_CLICK               |
+| threshold | float      | 1.1                        | Threshold that must be exceeded to be considered a click |
+| timeLimit | float      | 5                          | Max time in *ms* the acceleration can spend above the threshold to be considered a click |
+| latency   | float      | 10                         | Min time in *ms* between the end of one click event, and the start of another to be considred a DOUBLE_CLICK |
+| window    | float      | 50                         | Max time in *ms* between the start of one click event, and end of another to be considered a DOUBLE_CLICK |
+
+#### Single Click example
+```squirrel
+// Configure a single click interrupt
+accel.configureClickInt(true, LIS3DH.SINGLE_CLICK);
+```
+
+#### Double Click Example
+```squirrel
+// configure a double click interrupt
+accel.configureClickInt(true, LIS3DH.DOUBLE_CLICK);
+```
+
+### configureDataReadyInterrupt(*state*)
```

Comment:
This a "configure" method, while `setInterruptLatching()` is a "set" method, but they both just enable/disable a feature.  We should probably try to be consistent with naming here.